### PR TITLE
Attachments

### PR DESCRIPTION
**Pull request by inescoelho**
Add attachment feature.

### CODE REVIEW

File: lib/lotus/mailer/dsl.rb [hunk 1]
```diff
@@ -162,6 +163,23 @@ def subject(value = nil)
...
+      #
+      # @since 0.1.0
+      #
+      # @example
+      # class InvoiceMailer
+      #   include Lotus::Mailer
+      #
+      #   attachment 'path/to/file/attachment.pdf'
+      # end
+      def attachment(path)
```
Comment (joneslee85) [inline lib/lotus/mailer/dsl.rb]: what do you think about renaming this method to `attach` instead?

Comment (joneslee85) [inline lib/lotus/mailer/dsl.rb]: what do you think about allowing an array of attachment path as another option?

File: lib/lotus/mailer.rb [hunk 2]
```diff
@@ -66,7 +66,8 @@ def self.included(base)
         class_attribute :configuration
 
         self.configuration = conf.duplicate
-        self.templates(Hash.new)
+        self.templates = Hash.new
```
Comment (joneslee85) [inline lib/lotus/mailer.rb]: can we separate any changes made to `templates` to a separate PR?

Comment (rosafaria) [inline lib/lotus/mailer.rb]: We'll try :)

File: lib/lotus/mailer/dsl.rb (hunk 1 above)
Comment (rosafaria) [inline lib/lotus/mailer/dsl.rb]: Yeah, that's a good idea. We'll change that.

File: lib/lotus/mailer.rb (hunk 2 above)
Comment (rosafaria) [inline lib/lotus/mailer.rb]: I created another PR, please accept the other one before this one, the other has the templates changes without the attachment.

**Comment by joneslee85** [comment]
something not right, this PR get closed for no reason, can you please create a new PR for attach DSL? many thanks

**Pull request merged**